### PR TITLE
Update first-time-contributor.yml

### PR DESCRIPTION
**Pull request by aryannargotra**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [.] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/first-time-contributor.yml [hunk 1]
```diff
@@ -8,32 +8,33 @@ jobs:
...
+      (github.event.pull_request.author_association != 'CONTRIBUTOR') &&
+      (github.event.pull_request.author_association != 'MANNEQUIN') &&
+      (github.event.pull_request.author_association != 'MEMBER') &&
+      (github.event.pull_request.author_association != 'OWNER')
     steps:
       - name: Leave Welcome Comment
         uses: actions/github-script@v6
         with:
-          github-token: ${{ secrets.GH_ACCESS_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (leecalcote) [inline .github/workflows/first-time-contributor.yml:20]: There's no need to change the secret names. You'll want to revert this change.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks!

**Comment by vishalvivekm** [comment]
@aryannargotra 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT | 7:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: .github/workflows/first-time-contributor.yml [hunk 2]
```diff
@@ -8,32 +8,33 @@ jobs:
     if: >-
```
Comment (jerensl) [inline .github/workflows/first-time-contributor.yml:8]: it should be `>` only and without `-`

File: .github/workflows/first-time-contributor.yml [hunk 3]
```diff
@@ -8,32 +8,33 @@ jobs:
...
       (github.repository == 'meshery/meshery') &&
```
Comment (jerensl) [inline .github/workflows/first-time-contributor.yml:9]: are `( <expression> )` is valid syntax or you mean `${{ <expression> }}` as mentioned in the docs https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#about-expressions

**Review by jerensl — CHANGES_REQUESTED** [review]

File: .github/workflows/first-time-contributor.yml (hunk 3 above)
Comment (leecalcote) [reply .github/workflows/first-time-contributor.yml:9]: @aryannargotra are you incorporating this feedback?
